assets/dev/js/editor/controls/code.js

```diff
@@ -23,7 +23,7 @@ ControlCodeEditorItemView = ControlBaseDataView.extend( {
 
 		self.editor = ace.edit( this.ui.editor[ 0 ] );
 
-		jQuery( self.editor.container ).addClass( 'elementor-input-style elementor-code-editor' );
+		jQuery( self.editor.container ).addClass( 'elementor-input-style elementor-code-editor elementor-control-tag-area' );
```

Comment:
@nisan250 can you add explanation like this:
// Cover issue { here describe how the code fix it }

Reply:
Done.